Project.toml

```diff
@@ -9,13 +9,13 @@ DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 FrankWolfe = "f55ce6ea-fdc5-4628-88c5-0087fe54bd30"
+HiGHS = "87dc4568-4c63-4d18-b0c0-bb2238e4078b"
```

Comment:
do we need highs as main dependency? @dhendryc

Reply:
I think it is used the strong branching?

Reply:
ah yes indeed. It should be in the project toml already, strange